Bottom: c35d85df8539d60f2dfb5a92172da7b39b1197c3
Top:    d6042f47e890e2e27465e6eddcf69b8e0e9f7695
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-18 15:08:03 +0000

Refresh of attempt-to-refactor-the-step

---

diff --git a/src/hid/common/step_id.h b/src/hid/common/step_id.h
new file mode 100644
index 0000000..86a4974
--- /dev/null
+++ b/src/hid/common/step_id.h
@@ -0,0 +1,2 @@
+typedef int step_id;
+typedef GList* step_id_list;
